shell/config/product/harvester-manager.js

```diff
@@ -116,7 +116,7 @@ export function init(store) {
   ]);
   basicType([HCI.CLUSTER]);
   spoofedType({
-    label:      store.getters['i18n/t']('harvesterManager.cluster.label'),
+    labelKey:   'harvester.manager.cluster.label',
```

Comment:
This breaks the side nav menu item

Reply:
This looks like this to me:
![Screenshot 2022-12-12 at 9 34 01](https://user-images.githubusercontent.com/18264463/206999094-b73a260c-e5c8-4534-aa18-a80e5dda2916.png)
![Screenshot 2022-12-12 at 9 34 09](https://user-images.githubusercontent.com/18264463/206999098-0541414f-c60b-4168-b42e-0d263a5bfc20.png)